tests/specs/mcd/verification.k

```diff
@@ -252,4 +251,9 @@ module VERIFICATION
     // (check-sat)
     rule chop(X *Int Y) /sWord chop(Y) => X requires #rangeUInt(256, X) andBool #rangeSInt(256, Y) andBool #rangeSInt(256, X *Int Y) andBool Y =/=Int 0
 
+    // ### dsvalue-peek-pass-rough
+
+    rule (B ==K false) ==K false => B
```

Comment:
On Haskell this is not needed: https://github.com/kframework/evm-semantics/blob/master/tests/specs/functional/lemmas-no-smt-spec.k#L37
Because of this: https://github.com/kframework/evm-semantics/blob/fc44b9ddd1ba29f433778303e922ea73ab748df2/tests/specs/lemmas.k#L268-L269

Lemmas don't have side conditions, so they don't carry performance degradation risks. Should be made common for Java & Haskell.